docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,67 @@
+# Knative Eventing sprint planning
+
+This document summarizes project planning mechanics for Knative Eventing releases. Knative Eventing release is planned every six weeks or 6th Tuesday, with 0.5 planned on 04/02 as reference. Current proposal is to align sprint with the release cycle of 6 weeks. Eventually this may change as we learn from the process.
+
```

Comment:
I strongly suggest having at this section a table/chart/diagram that explains in a blink the planning timeline and "ceremonies" that will take place (proposal submitting, triage, snapshot, etc etc) as @evankanderson mentioned below as well.